EXM105/Makefile

```diff
@@ -21,6 +21,21 @@ connectathon:
   -H 'Content-Type: application/json' \
 	-d @connectathon/fhir3/bundles/EXM105_FHIR3-8.0.000/EXM105_FHIR3-8.0.000-bundle.json
 
+.seed-cqf-ruler-no-vs:
+	until `curl --output /dev/null --silent --head --fail http://localhost:8080/cqf-ruler-dstu3`; do printf '.'; sleep 5; done
```

Comment:
I think I might defer on this one until the task that refactors all the Makefiles. As it is now, we still want to be able to run standalone `make`s from the individual measure directories, so I think in the meantime this might need to stay where it is.

Agree that it could be moved in the future thought.